### PR TITLE
[ chore ] Australian Dollar support added in Create Notion Table

### LINKED ISSUE #42: [Chore]  Australian dollar Number Format support in Create Table Feature and Some Chores
### What happened?

- There has been some changes in notion api: 
FInd complete change log: https://developers.notion.com/page/changelog

- New ID Field introduced in Notion Table, check if that breaks anything in our app.

- The [verification](https://developers.notion.com/reference/page-property-values#verification) page property was added to the [Page properties](https://developers.notion.com/reference/page-property-values) documentation. see if it breaks and we can improve something.

## [ChangeLog](https://developers.notion.com/page/changelog#september-6---september-7-2023) September 6 - September 7, 2023
- The [updates](https://developers.notion.com/page/changelog#notice-for-an-upcoming-public-api-change) related to the [Formulas 2.0 launch](https://twitter.com/NotionHQ/status/1699828805408550971?s=20) are now live in the Public API. These changes will not impact most developers using the Public API; however, please note that the formatting of [formula.expression](https://developers.notion.com/reference/property-object#formula), which is returned when [retrieving a database](https://developers.notion.com/reference/retrieve-a-database) with a [Formula property](https://developers.notion.com/reference/property-object#formula), has changed. See Notion's Help Center articles for more information on the Formula 2.0 changes:
[Formulas 2.0: How to use Notion’s new and improved formulas with your existing setups](https://www.notion.so/help/guides/new-formulas-whats-changed)
[How to write Notion formulas that extend the capabilities of your databases](https://www.notion.so/help/guides/write-formulas-that-extend-capabilities-of-databases)
- The example for the [Formula database property](https://developers.notion.com/reference/property-object#formula) was updated to align with the new Formula 2.0 launch.
- also read: https://developers.notion.com/page/changelog#august-8---august-22-2023

### Steps to reproduce

- [x] Check if the formula property works as expected in /notion create db. and make changes accordingly.
- [x] as australian dollar has been supported now. under /notion create db if the Notion table includes a number then it should have a number format of australian dollar too. [Check SS below]

### Relevant ScreenShots

Create

### Version

v6.5

### What browsers are you seeing the problem on?

Chrome

### PR DESCRIPTION
**Pull request by Sayan4444**
## Proposed changes (including videos or screenshots)

CHANGES MADE

1. I have checked multiple formulas and they did give me correct answers, if there is any exception please do inform me
2. I have added support for Australian Dollars. 

## Issue(s)

This should close issues :-
fix #42 

## Steps to test or reproduce

 1. Check if the formula property works as expected in /notion create db. and make changes accordingly.
 2. as australian dollar has been supported now. under /notion create db if the Notion table includes a number then it should have a number format of australian dollar too.

### CODE REVIEW

**Comment by Sayan4444** [comment]
Hi @Nabhag8848 if I did any wrong or misunderstood something, please do inform me. I will love to rectify myself

File: package-lock.json
Comment (Nabhag8848) [inline package-lock.json:1]: Can you revoke this file changes in local and push again. currently we don't need to have updated lock file changes in main.

File: package.json
Comment (Nabhag8848) [inline package.json:1]: Question: is there any updates in apps engine ? cause as long as i believe we don't need to install @rocket.chat/ui-kit in dependecies to let uikit work.

**Review by Nabhag8848 — CHANGES_REQUESTED** [review]
Changes looks good, fixes the issue. added some comments have a look and also as this is small PR would you like to take up 
- #12 
and extend this. its beginner level and you could figure it out easily as i believe there is already similar code.

File: package.json
Comment (Sayan4444) [reply package.json:1]: I have been facing error and npm deploy was not running

Comment (Nabhag8848) [reply package.json:1]: which version of RC you are using ?

File: package-lock.json
Comment (Sayan4444) [reply package-lock.json:1]: Yes bhaiya I did it

File: package.json
Comment (Sayan4444) [reply package.json:1]: image

**Comment by Sayan4444** [comment]
> Changes looks good, fixes the issue. added some comments have a look and also as this is small PR would you like to take up
> 
> * [[Feat] Add Change Workspace Option in Overflow Menu within Create Page or Record Modal #12](https://github.com/RocketChat/Apps.Notion/issues/12)
>   and extend this. its beginner level and you could figure it out easily as i believe there is already similar code.

Sure, soon will start working in this issue

**Comment by Nabhag8848** [comment]
> > Changes looks good, fixes the issue. added some comments have a look and also as this is small PR would you like to take up
> > 
> > * [[Feat] Add Change Workspace Option in Overflow Menu within Create Page or Record Modal #12](https://github.com/RocketChat/Apps.Notion/issues/12)
> >   and extend this. its beginner level and you could figure it out easily as i believe there is already similar code.
> 
> Sure, soon will start working in this issue

LGTM, Lemme know if you have any questions around #12

**Comment by Sayan4444** [comment]
Yes @Nabhag8848 bhai I have a question. How can we track that the workspace has been opened from 'OverFlowComponent' and not from '/notion create db'. By using that only I will conditionally return the modal to 'pages create modal'

**Comment by Nabhag8848** [comment]
Thanks @Sayan4444 Good start! Congrats! Just letting you aware, as this is chore, won't be shown in gsoc leaderboard.

**Pull request merged**